packages/@react-stately/layout/src/ListLayout.ts

```diff
@@ -147,6 +147,7 @@ export class ListLayout<T> extends Layout<Node<T>> implements KeyboardDelegate,
     return res;
   }
 
+  // TODO: can have isVisible just return true instead of mocking the visible rect, but virtualizer mocks are more centralized
```

Comment:
Are you asking for ideas about or feedback on the TODO comments? These are still problems, right?

Reply:
ideas/feedback welcome here. This specific todo is marking an alternative method from the virtualizer changes I've made to get all items to render in test, but I'm open to any pros/cons/alternatives to either approach